MobileWallet/Common/Extensions/UIColor+App.swift

```diff
@@ -0,0 +1,72 @@
+//  UIColor+App.swift
+
+/*
+	Package MobileWallet
+	Created by S.Shovkoplyas on 17.04.2020
+	Using Swift 5.0
+	Running on macOS 10.15
+
+	Copyright 2019 The Tari Project
+
+	Redistribution and use in source and binary forms, with or
+	without modification, are permitted provided that the
+	following conditions are met:
+
+	1. Redistributions of source code must retain the above copyright notice,
+	this list of conditions and the following disclaimer.
+
+	2. Redistributions in binary form must reproduce the above
+	copyright notice, this list of conditions and the following disclaimer in the
+	documentation and/or other materials provided with the distribution.
+
+	3. Neither the name of the copyright holder nor the names of
+	its contributors may be used to endorse or promote products
+	derived from this software without specific prior written permission.
+
+	THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND
+	CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+	INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+	OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+	DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+	CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+	SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+	NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+	LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+	HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+	CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+	OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+	SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+import UIKit
+
+extension UIColor {
+
+    // MARK: - Gradient -
+
+    static var auroraGradient1: UIColor { return UIColor(named: "auroraGradient1") ?? UIColor(red: 98.0 / 255.0, green: 57.0 / 255.0, blue: 1.0, alpha: 1.0)
```

Comment:
This can also move to the theme file. 
Also if you add these colors to the theme file there will be a unit test that enforces all color assets are in the project. So you don't need to provide a fallback color, you can even force unwrap them when you need to.
https://github.com/tari-project/wallet-ios/blob/689f586924efc4871ad2b99b1be77b7fa8f61c5f/MobileWalletTests/MobileWalletTests.swift#L55